src/include/btmem.h

```diff
@@ -1253,9 +1257,9 @@ struct __wt_ref {
  * WT_REF_SIZE is the expected structure size -- we verify the build to ensure the compiler hasn't
  * inserted padding which would break the world.
  */
-#define WT_REF_SIZE (48 + WT_REF_SAVE_STATE_MAX * sizeof(WT_REF_HIST) + 8)
+#define WT_REF_SIZE (56 + WT_REF_SAVE_STATE_MAX * sizeof(WT_REF_HIST) + 8)
```

Comment:
It's a concern that we increase 1 byte for each ref in memory.

Reply:
The other way is to add WT_REF_FLAG_INTERNAL and WT_REF_FLAG_LEAF to the flag_atomics as well.